interface/index.d.ts

```diff
@@ -0,0 +1,22 @@
+import { NextApiRequest } from 'next';
+import { Auth } from './auth';
+
+export interface NextApiRequestQueryBody<TQuery, TBody> extends NextApiRequest {
```

Comment:
Why does this look exactly like the previous file? And not sure I understand the point of these interfaces,

Reply:
This is a placeholder and not used.